FeatureTest.java

```diff
@@ -6,7 +6,8 @@ public static void main(String[] args) {
     public int square(int variable){
         String str = null;
         System.out.println(str.length());
-        str = "fjff";
+        String newer = null;
+        System.out.println(newer.length());
         return variable * variable;
```

Comment:
Use a logger (e.g., SLF4J) with ERROR level and include exception details: logger.error("Null pointer exception occurred", e);